README.md

```diff
@@ -1,11 +1,99 @@
 # Load Secrets from 1Password - GitHub Action
 
-The action to load secrets from [1Password Connect](https://1password.com/secrets/) into GitHub Actions.
+This action loads secrets from [1Password Connect](https://1password.com/secrets/) into GitHub Actions.
+
 
 Specify right from your workflow YAML which secrets from 1Password should be loaded into your job, and the action will make them available as environment variables for the next steps.
 
+## Prerequisites 
+ - [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) deployed in your infrastructure
```

Comment:
This is no longer a prerequisite, since it only applies when the users want to use the action with Connect.

Reply:
changed